### ISSUE TITLE
Please stop using "starring" in npm postinstall

### ISSUE THREAD

**Issue by zech** [ritz078/rollup-plugin-filesize#15]
Hi,

I'd like not to be forced to star a package from npm providing Github's username, password as well as second factor authentication.

If you want to keep [`starring`](https://github.com/ritz078/starring) in your package.json please add the ability to skip the process without failing the install.

```
> rollup-plugin-filesize@1.3.1 postinstall C:\Users\<me>\Documents\<some path>\node_modules\rollup-plugin-filesize
> starring rollup-plugin-filesize

Your GitHub username:
```

**Comment by ritz078** [comment]
Hey sorry for the bad experience. I definitely didn't do that to get stars. I was actually making a demo gif and for that I made a beta release (It wasn't intended to be used by anyone but only for my personal purpose of creating a video to ask the users' feedback). But I see that npm messed it up and made a proper release instead of beta release.
Sorry for that. Fixed in v1.3.1.

**Comment by zech** [comment]
Thank you!
